booking-engine-api/operations/README.md

```diff
@@ -7,6 +7,7 @@ This section describes all operations supported by the API, organised by theme.
 | <div style="width:200px">Operation or Endpoint</div> | Description |
 | :-- | :-- |
 | [Get availability](hotels.md#get-availability) | Get availability and pricing options for the specified hotel for the specified date interval |
+| [Get availability blocks](availability-blocks.md#get-availability-blocks) | Get details for the given availability block for the given hotel |
```

Comment:
This should be 'blocks' (plural). On reflection, it might be a bit easier to read to say simply "Get details of the availability blocks for the given hotel."